### ISSUE TITLE
Word doesn't work again with 5.49.7

### ISSUE THREAD

**Issue by Seeadler1** [sandboxie-plus/Sandboxie#773]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**System details and installed software (please provide the following information):**
 - What is your Windows edition and version? (i.e. Windows 10 Pro 20H2).
 - What is your current Sandboxie edition and version? (i.e. Sandboxie Plus 0.7.2).
 - Please mention in which version this bug got introduced, because it acts as a guide for testers and developers.
 - Please mention any security software running in the background, even if the real-time protection is disabled.

**Additional context**

**Sandboxie configuration**
If applicable, consider to attach your sandboxie.ini configuration by copying the content at https://gist.github.com and sharing the resulting link here. If you decide to paste the configuration directly here, make sure to use the backticks around strings, like in this working example:

<details>
 
<summary>My initial sandboxie.ini settings</summary>
 
```

[GlobalSettings]

.....

[UserSettings_175D0429]

.....

[DefaultBox]

.....

```

</details>

**Comment by Seeadler1** [comment]
Word has stopped working since 5.49.7. This problem did not exist with the 5.49.5

Regards

**Comment by isaak654** [comment]
You're right, issue #428 is back again. It only affects the Click to Run version, but I don't think it's Sandboxie fault. For example, downgrading from 0.7.4 to 0.7.3 didn't fix the issue for me.

![error_word_c2r_version](https://user-images.githubusercontent.com/12372772/114314012-4465f980-9af9-11eb-979d-bb491bb262c4.png)

**Comment by APMichael** [comment]
I can confirm that it has nothing to do with Sandboxie, as it did last time. Microsoft has broken the functionality with Sandboxie again with the current Office update. 😞 The beta version had already hinted at this... We can only hope that @DavidXanatos will eventually figure out the changes and update Sandboxie accordingly.

**Comment by Seeadler1** [comment]
Thank you Michael!!!

**Comment by patoniilista** [comment]
I'm using sandboxie plus 0.7.5 and was having the same problem. I managed to fix it by adding the whole Word installation folder to the _Forced Programs_ Sandboxie section, instead of trying to install inside Sandboxie or opening the exe through the explorer context menu. This has been working great for me since 0.7.2
![image](https://user-images.githubusercontent.com/72221896/120939409-570e4080-c6ee-11eb-8f86-a36bf82d2204.png)
Now not only Word works, but Excel etc does too.
![image](https://user-images.githubusercontent.com/72221896/120939876-8c1b9280-c6f0-11eb-92c4-04f14ee44b02.png)

I also have _Drop rights from Administrators and Power Users groups_ and _Make applications think they are running elevated (allows to run installers safely)_ both checked, if that helps.

Ps. @DavidXanatos there's a typo in the last checkbox I mentioned. Unless you're talking about https://forgottenrealms.fandom.com/wiki/Thay :)

![image](https://user-images.githubusercontent.com/72221896/120939745-d05a6300-c6ef-11eb-8c55-736f396f7c83.png)

**Comment by APMichael** [comment]
@DavidXanatos: Are there any new findings on this yet? Can the issue be solved soon?

The workaround may work, but it is only useful for users who always want to open Office sandboxed (because of ForceProcess/ForceFolder).

However, many users only want to sandbox Office on demand.

**Comment by DavidXanatos** [comment]
no updates yet

**Comment by APMichael** [comment]
> no updates yet

Sad news. Thanks for your reply anyway.

**Comment by tienlhp1** [comment]
sad news with me.

**Comment by isaak654** [comment]
Duplicate of https://github.com/sandboxie-plus/Sandboxie/issues/428, please post there if you have updates.

**Comment by isaak654** [comment]
It was fixed on v0.9.7 / 5.52.1: https://github.com/sandboxie-plus/Sandboxie/releases/tag/0.9.7
